### PR TITLE
Feature/frontend/sign email success

### PR DESCRIPTION
**Pull request by urbaniakmichal**

### CODE REVIEW

File: frontend/src/app/app-contact/contact-form/contact-form.component.ts [hunk 1]
```diff
@@ -57,6 +57,8 @@ import {ContactFormService} from "./contact-form.service";
           mat-button
           [disabled]="!contactFormService.form.valid">Send
         </button>
+        <div>
```
Comment (leoni-q) [inline frontend/src/app/app-contact/contact-form/contact-form.component.ts]: remove it

File: frontend/src/app/app-contact/contact-form/contact-form.service.ts [hunk 2]
```diff
@@ -35,7 +37,15 @@ export class ContactFormService {
 
     this.http
       .post('/api/skill-tree/v1/contact/request', formData)
-      .subscribe((error) => console.log(error))
+      .subscribe(
+        () => {
```
Comment (leoni-q) [inline frontend/src/app/app-contact/contact-form/contact-form.service.ts:41]: 1. Thanks for your email :)
2. remove console.log
3. There is a problem with sending your email

File: frontend/src/app/utils/notification/notification.service.ts [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import {Injectable, NgZone} from '@angular/core';
+import {MatSnackBar} from "@angular/material/snack-bar";
+
+@Injectable({
+  providedIn: 'root'
+})
+export class NotificationService {
+
+  constructor(
+    public snackBar: MatSnackBar, private zone: NgZone) { }
```
Comment (leoni-q) [inline frontend/src/app/utils/notification/notification.service.ts]: can snackBar be private?

File: frontend/src/app/utils/errors/globalErrorHandler.ts [hunk 4]
```diff
@@ -0,0 +1,31 @@
+import {ErrorHandler, Injectable, Injector} from '@angular/core';
+import {HttpErrorResponse} from "@angular/common/http";
+import {ErrorService} from "./error.service";
+import {ConsoleLoggingService} from "../logging/consoleLogging.service";
+import {NotificationService} from "../notification/notification.service";
+
+@Injectable()
+export class GlobalErrorHandler implements ErrorHandler {
```
Comment (leoni-q) [inline frontend/src/app/utils/errors/globalErrorHandler.ts]: change file name to global-error-handler.ts

File: frontend/src/app/utils/logging/consoleLogging.service.ts [hunk 5]
```diff
@@ -0,0 +1,13 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ConsoleLoggingService {
```
Comment (leoni-q) [inline frontend/src/app/utils/logging/consoleLogging.service.ts]: is this necessary? We actually log error to console in globalErrorHandler

**Review by leoni-q — CHANGES_REQUESTED** [review]

File: frontend/src/app/utils/logging/consoleLogging.service.ts (hunk 5 above)
Comment (urbaniakmichal) [reply frontend/src/app/utils/logging/consoleLogging.service.ts]: hmmm its nice look xd

File: frontend/src/app/app.module.ts [hunk 6]
```diff
@@ -26,6 +25,10 @@ import {HeroActionButtonsComponent} from './landing-page/top/hero-action-buttons
 import {AppContactComponent} from "./app-contact/app-contact.component";
 import {ContactFormComponent} from "./app-contact/contact-form/contact-form.component";
 import {ContactMapComponent} from "./app-contact/contact-map/contact-map.component";
+import {NotificationComponent} from './utils/notification/notification.component';
```
Comment (urbaniakmichal) [inline frontend/src/app/app.module.ts]: notification.component is not exist so i removed it line

**Review by leoni-q — APPROVED** [review]

**Pull request merged**